### PR TITLE
English spellcheck

### PR DESCRIPTION
**Pull request by Shdorsh**
Spellchecks a few files in the english version.

### CODE REVIEW

File: system/lang/en/attribute.txt [hunk 1]
```diff
@@ -3,31 +3,31 @@ relatedAttributes = "Related attributes"
...
+	desc "\tIncreases the damage dealt in melee combat, the maximum carry capacity and health slightly. Required for the efficient use of heavy items.\n\tUsing heavy weapons and armor or carrying heavy items trains strength.\n\tAffected skills: one-handed weapons, long blades, blunt weapons, axes, unarmed, shields, heavy armor."
 }
 attribute end {
 	name "Endurance"
-	//desc "\tIncrease health, stamina and resistance.\n\tTaking damage and natural healing trains endurance.\n\tAffects skills: medium armor, heavy armor, concentration, survival, rage."
-	desc "\tIncrease health, stamina and resistance.\n\tTaking damage and natural healing trains endurance.\n\tAffects skills: medium armor, heavy armor."
+	//desc "\tIncreases health, stamina and resistance.\n\tTaking damage and natural healing trains endurance.\n\tAffected skills: medium armor, heavy armor, concentration, survival, rage."
+	desc "\tIncreases health, stamina and resistance.\n\tTaking damage and natural healing trains endurance.\n\tAffected skills: medium armor, heavy armor, concentration."
```
Comment (Tomash667) [inline system/lang/en/attribute.txt]: Concentration skill is not implemented yet so it was only inside comment.

File: system/lang/en/attribute.txt [hunk 2]
```diff
@@ -3,31 +3,31 @@ relatedAttributes = "Related attributes"
...
-	desc "\tIncrease the damage dealt in melee combat, carry capacity and slightly health. Required for the efficient use of heavy items.\n\tUsing heavy weapons and armor or caring heavy items trains strength.\n\tAffects skills: one-handed weapon, long blade, blunt, axe, shield, heavy armor."
+	//desc "\tIncreases the damage dealt in melee combat, the maximum carry capacity and health slightly. Required for the efficient use of heavy items.\n\tUsing heavy weapons and armor or carrying heavy items trains strength.\n\tAffected skills: one-handed weapons, long blades, blunt weapons, axes, unarmed, shields, heavy armor, athletics."
+	desc "\tIncreases the damage dealt in melee combat, the maximum carry capacity and health slightly. Required for the efficient use of heavy items.\n\tUsing heavy weapons and armor or carrying heavy items trains strength.\n\tAffected skills: one-handed weapons, long blades, blunt weapons, axes, unarmed, shields, heavy armor."
```
Comment (Tomash667) [inline system/lang/en/attribute.txt]: Unarmed combat skill is not implemented yet.

**Comment by Tomash667** [comment]
Thank you!

**Comment by Shdorsh** [comment]
No problem. I'd like to help in coding stuff aswell, since I believe this game could have some potential, but I don't know any C++. I'd have to get into it slowly.

**Pull request merged**